File: LINEAR_SEARCH.cpp
#include<iostream>
using namespace std;
int main()
{
    int a[100];
    int size, i, j;
    int flag = 0;
    cout<<"Enter the Size of Array: ";
    cin>>size;
    cout<<"\nEnter the Elements of Array one by one";
    for(i=0; i<size; i++)
    {
        cout<<"\nEnter Element as "<<i+1<<" position: ";
        cin>>a[i];
    }

    int item;
    cout<<"\nEnter the Item which you want to search: ";
    cin>>item;

    for(i=0; i<size; i++)
    {
        if(a[i] == item)
        {
            flag = 1;
            break;
        }
    }

    if(flag == 1)
    cout<<"\nElement found at "<<i+1<<" position";

    else
    cout<<"\nElement Not Found at any of Position in Array"<<"\nEnter Correct Element";

   return 0;
}

File: Algorithms/yourname.github/LinearSearch.java
import java.util.Scanner;

public class LinearSearch{

public static void main(String [] args){

int x;
int a[]=new int[100];
boolean found=false;
Scanner scan=new Scanner(System.in);

System.out.println("enter the size of the elements");
int n=scan.nextInt();
System.out.println("enter "+n+" elements ");
for(int i=0;i<n;i++)
a[i]=scan.nextInt();

System.out.println("enter the  elements to find in list");
x=scan.nextInt();


for(int i=0;i<n;i++)
{
	if(a[i]==x)
	{
	System.out.println("element found at position:\t"+(i+1));
	found=true;
	 break;
	}
}

if(!found)
System.out.println("element not found in list");

}

}